src/server/Adaptive.ReactiveTrader.Server.Pricing/Adaptive.ReactiveTrader.Server.Pricing.csproj

```diff
@@ -18,6 +18,7 @@
   </ItemGroup>
 
   <ItemGroup>
+    <PackageReference Include="HtmlAgilityPack" Version="1.11.12" />
```

Comment:
Agree with this. If we have price sources that are through an API, why do we need this at all?

Reply:
This is only the fourth fall-back adapter..
First: EuropeanCentralBankAdapter (json API)
Second: FinancialModelingPrepAdapter (json API)
Third: Fixer.IO (json API not enabled yet because we do not have an API key yet)
Fourth: Yahoo.com (html scrape)

Reply:
Need to discuss. Is RT a real time trading platform to demonstrate to potential clients (Amanda), or is RT a trading platform demo showcasing reactive programming principles applied across the full application stack (the root readme.md) . IMHO web scrapes are nasty but they are a necessary evil in the real world.